dashboard/src/modules/components/AuthComponent/common-components.jsx

```diff
@@ -80,46 +75,25 @@ export const LoginRightComponent = () => {
 
 export const AuthForm = () => {
   const navigate = useOutletContext();
+  const { keycloak } = useKeycloak();
 
   const dispatch = useDispatch();
   const navigatePage = (toPage) => {
     dispatch(movePage(toPage, navigate));
   };
+  useEffect(() => {
+    dispatch(authCookies());
+  }, );
   return (
     <Card className="auth-card">
       <CardTitle>
         <LoginHeader title="Login with..." />
       </CardTitle>
-      <CardBody>
-        <div className="button-wrapper">
-          <Button
-            variant="primary"
-            onClick={() => navigatePage(APP_ROUTES.AUTH_LOGIN)}
-          >
-            Pbench Credentials
-          </Button>
-        </div>
-        <div className="account-wrapper">
-          <div>
-            <span>Need an account?</span>
-            <Button
-              variant="link"
-              onClick={() => navigatePage(APP_ROUTES.AUTH_SIGNUP)}
-            >
-              Sign up
-            </Button>
```

Comment:
Do we have something equivalent to this which helps people find the OIDC login/account-creation?

Reply:
That would be `http://<keycloak_host>:<keycloak_port>/realms/<realm-name>/account/#/`

Reply:
OK, but does the Dashboard present that (or a nice button which references it) to the user anywhere?  Or, do we just assume that the user will know how to obtain an account?  (Or, do we figure that the user will click on the "login" button (even if they have no account) and that the OIDC provider's page will help them out?)

Reply:
> OK, but does the Dashboard present that (or a nice button which references it) to the user anywhere? Or, do we just assume that the user will know how to obtain an account? (Or, do we figure that the user will click on the "login" button (even if they have no account) and that the OIDC provider's page will help them out?)

We would not have a user registration on our Keycloak broker. Our keycloak instance (Intlab/our own) will only have a list of identity providers to choose from. Once the broker redirects the user to the identity provider's page it's up to the identity provider's configuration to give the user an option for login and signup.

Reply:
Right, but think about it from a user's perspective:  your coworker shares a URL with you, which loads the Pbench Dashboard in non-logged-in mode; the page text leads you to the auth page which says something like "click here to log in"; however, you're not sure you have a login, and so you're trying to figure out what to do.

I think the answer might be to carefully craft the text on the auth page to explain or convey the fact that the user can log in using whichever of the supplied identity providers they have an account with and that, if they wish to create an account, they should go to the identity provider's page.  And, we need to say it in much friendlier words than those....

My concern is that your change here seems like it has removed all of the friendliness and guidance from this page and replaced it with a simple "click this button".

Reply:
I think the question is how much information about the Keycloak/OIDC configuration we have available in the dashboard to be "helpful" here. Maybe?

Reply:
I think all we need to do is say something like, "Here are the services that you can use to log in to the Pbench Server; if you need to create an account on the service-provider that you would like to use, please contact that service provider".

Reply:
But if the user sees the list of identity providers they should know whether they have an account on any of them if not they should already know how to create one separately. I believe all the well-known identity providers provide an option to create an account when they give the option to login.